src/index.html

```diff
@@ -5,9 +5,13 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Wbpack Exercise</title>
```

Comment:
just a suggestion please update the project title to accurately reflect the goals and objectives of your project.